ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/data/PaywallViewModel.kt

```diff
@@ -152,11 +147,6 @@ internal class PaywallViewModelImpl(
         }
     }
 
-    override fun openURL(url: URL, context: Context) {
```

Comment:
There is one downside of moving this outside the viewModel, which is that if we want to have events or some other logic when navigating to links, having it as part of the compose tree might not be as easy to test or handle... But we can do that later if we have to.